no `Alloc` in `alloc`

```
   Compiling bitflags v1.2.1
   Compiling lazy_static v1.4.0
   Compiling linked_list_allocator v0.6.5
error[E0432]: unresolved import `alloc::alloc::Alloc`
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/linked_list_allocator-0.6.5/src/lib.rs:14:20
   |
14 | use alloc::alloc::{Alloc, AllocErr, Layout};
   |                    ^^^^^
   |                    |
   |                    no `Alloc` in `alloc`
   |                    help: a similar name exists in the module: `alloc`

error: aborting due to previous error
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -11,7 +11,7 @@ extern crate spin;
 
 extern crate alloc;
 
-use alloc::alloc::{Alloc, AllocErr, Layout};
+use alloc::alloc::{AllocRef, AllocErr, Layout};
 use core::alloc::{GlobalAlloc};
 use core::mem;
 #[cfg(feature = "use_spin")]
@@ -129,7 +129,7 @@ impl Heap {
     }
 }
 
-unsafe impl Alloc for Heap {
+unsafe impl AllocRef for Heap {
     unsafe fn alloc(&mut self, layout: Layout) -> Result<NonNull<u8>, AllocErr> {
         self.allocate_first_fit(layout)
     }

